frontend/src/components/CanteenPublication.vue

```diff
@@ -1,5 +1,51 @@
 <template>
   <div class="text-left">
+    <v-col v-if="canteen && canteen.publicationComments && !editDescription" cols="12" sm="6" class="px-0">
+      <h2 class="fr-text grey--text text--darken-4 mb-6">
+        Description de l'établissement
+      </h2>
+      <div class="ml-n8">
+        <DsfrHighlight>
+          <p>
+            {{ canteen.publicationComments }}
+          </p>
+        </DsfrHighlight>
+      </div>
+      <v-btn
+        v-if="editable"
+        @click="
+          editDescription = true
+          oldPublicationComments = canteen.publicationComments
+        "
+        outlined
+        small
+        color="primary"
+        class="fr-btn--tertiary px-2 mt-4"
+      >
+        <v-icon primary x-small class="mr-1">mdi-pencil-outline</v-icon>
+        Modifier la description
+      </v-btn>
+    </v-col>
+    <v-col v-else-if="canteen && editable" cols="12" sm="6">
+      <v-form v-model="publicationFormIsValid" ref="publicationCommentsForm">
+        <DsfrTextarea
+          class="mt-2"
+          rows="5"
+          counter="500"
+          v-model="canteen.publicationComments"
+          :rules="[validators.maxChars(500)]"
+        >
+          <template v-slot:label>
+            <span class="fr-label mb-1">Déscription de l'établissement</span>
+            <span class="fr-hint-text mb-2">
+              Si vous le souhaitez, personnalisez votre affiche en écrivant quelques mots sur votre établissement : son
```

Comment:
On a le break du `:` en pleine ligne
![image](https://github.com/betagouv/ma-cantine/assets/1225929/258cd2e7-74c7-436c-82b5-f65eddf5b2b0)
```suggestion
              Si vous le souhaitez, personnalisez votre affiche en écrivant quelques mots sur votre établissement&nbsp;: son
```